cfgov/jinja2/v1/_includes/organisms/form-explainer.html

```diff
@@ -0,0 +1,198 @@
+{# ==========================================================================
+
+   render_map_points( categories, page_num )
+
+   ==========================================================================
+
+   Description:
+
+   Render a list of o-expandable notes with definitions.
+
+   categories:  An array of category objects containing note arrays.
+
+   page_num: The page that these map points belong to.
+
+   ========================================================================== #}
+
+{% macro render_map_points( categories, page_num ) %}
+{% for category in categories %}
+    {% set category_index = loop.index %}
+    {% for item in category.notes %}
+        {% set id = 'page-' + page_num | string + '-category-' + category_index | string + '-note-'  + loop.index | string %}
+        <a class="image-map_overlay image-map_overlay__category-{{ category_index }}"
+           style="{{ 'top:' + item.coordinates.top | string + '%; left:' + item.coordinates.left | string + '%; width:' + item.coordinates.width | string + '%; height:' + item.coordinates.height | string + '%;' }}"
+           href="#{{ id }}"
+           id="image-map_overlay__{{ id }}"
+           tabindex="-1">
+          <span class="u-visually-hidden">{{ item.heading }}</span>
+         </a>
+    {% endfor %}
+{% endfor %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_terms( terms, page_num )
+
+   ==========================================================================
+
+   Description:
+
+   Render a list of o-expandable terms with definitions.
+
+   notes:    An array of note objects.
+
+   page_num: The page that these notes belong to.
+
+   category_index: category that these notes belong to.
+
+   ========================================================================== #}
+
+{% macro render_terms( notes, page_num, category_index ) %}
+{% for item in notes %}
+    {{ render_term( item, page_num, category_index, loop.index ) }}
+{% endfor %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_term( note, page_num, category_index, note_index )
+
+   ==========================================================================
+
+   Description:
+
+   Render an o-expandable note with a definition.
+
+   item:      A note object. Note objects have the following structure:
+              {
+                'heading': 'Note title.',
+                'body':    'Text of note.'
+              }
+
+   page_num:   Page containing note.
+
+   category_index: Category containing note.
+
+   note_index: index of note in category note array.
+
+
+   ========================================================================== #}
+
+{% macro render_term( item, page_num, category_index, note_index ) %}
+{% set id = 'page-' + page_num | string + '-category-' + category_index | string + '-note-'  + note_index | string %}
```

Comment:
Would it be possible to pull in the expandable macro here, to avoid duplication? Would it be preferable to, say, add the ability to add extra classes [here](https://github.com/cfpb/cfgov-refresh/blob/f31b1c31439aae237744a2805a3c812218fde201/cfgov/jinja2/v1/_includes/organisms/expandable.html#L37) via another macro parameter, instead of duplicating?

Reply:
@chosak This will probably happen in a forthcoming front-end updates PR, once I clarify a tabindex issue.